### ISSUE TITLE
False positive on `ExplicitCollectionElementAccessMethod` with spread operator

### ISSUE THREAD

**Issue by BraisGabin** [detekt/detekt#5640]
## Expected Behavior
I have this interface:
```kotlin
interface Foo {
    operator fun get(key: String, vararg objects: Any): String
}
```

And when I call it like this `foo.get(key, *objects)` I expect that `ExplicitCollectionElementAccessMethod` will not flag it. `foo[key, *objects]` is not valid kotlin code so the only way to call it is `foo.get(key, *objects)`

## Observed Behavior
`foo.get(key, *objects)` is flagged

## Your Environment
* Version of detekt used: 1.22.0

**Comment by cortinico** [comment]
`ExplicitCollectionElementAccessMethod` should probably by extended to ignore `varargs`

**Comment by atulgpt** [comment]
Hi @BraisGabin/ @cortinico I have create a PR https://github.com/detekt/detekt/pull/5654/files to fix this issue. Let me know if the fix looks good. Thanks
